test/server/http/admin_test.cc

```diff
@@ -13,13 +14,17 @@ class AdminFilterTest : public testing::Test {
 public:
   // TODO(mattklein123): Switch to mocks and do not bind to a real port.
   AdminFilterTest()
-      : admin_("/dev/null", Network::Utility::resolveUrl("tcp://127.0.0.1:9002"), server_),
+      : admin_("/dev/null", "/tmp/envoy.prof", Network::Utility::resolveUrl("tcp://127.0.0.1:9002"),
+               server_),
+        admin_bad_profiler_path_("/dev/null", "/var/log/envoy/envoy.prof",
+                                 Network::Utility::resolveUrl("tcp://127.0.0.1:9002"), server_),
         filter_(admin_), request_headers_{{":path", "/"}} {
     filter_.setDecoderFilterCallbacks(callbacks_);
   }
 
   NiceMock<MockInstance> server_;
   AdminImpl admin_;
+  AdminImpl admin_bad_profiler_path_;
```

Comment:
I don't think this needs to be a member of `AdminFilterTest`, it can be just allocated and instantiated locally in `AdminBadProfiler`.